wasm/src/main/scala/wasm4s/WasmContext.scala

```diff
@@ -212,7 +213,10 @@ class WasmContext(val module: WasmModule) extends TypeDefinableWasmContext {
   private val _importedModules: mutable.LinkedHashSet[String] =
     new mutable.LinkedHashSet()
 
-  override protected var nextItableIdx: Int = 0
+  def buckets: List[Bucket] = _buckets
+  private var _buckets: List[Bucket] = Nil
+  def assignBuckets(classes: List[LinkedClass]): Unit =
+    _buckets = assignBuckets0(classes)
```

Comment:
Perhaps immediately filter out all classes that are `.kind.isJSType`. It is a bit hard to reason about what the algorithm will do with JS types, as is, but in fact it shouldn't do anything at all.